src/main/scala/com/mongodb/spark/sql/MongoInferSchema.scala

```diff
@@ -59,17 +59,22 @@ object MongoInferSchema extends Logging {
    * @return the schema for the collection
    */
   def apply(mongoRDD: MongoRDD[BsonDocument]): StructType = {
```

Comment:
@ganeshayadiyala Lets have a different branch. 2.2.0.1 or 2.2.1 to incorporate this change.